src/angular-bootstrap-toggle.js

```diff
@@ -1,289 +1,337 @@
-(function () {
-    'use strict';
+(function() {
+	"use strict";
+	if (window.IntersectionObserver) {
+		$.event.special.visibility = {
```

Comment:
The problem is that we would now require jQuery for the `<toggle>` directive to work, which is something that is usually not wanted in AngularJS (where it is often recommended to remove jQuery when ever possible).

Would this be feasible with `angular.element()` somehow?

Reply:
yes ... jquery removed